### PR TITLE
Update consumer deployment and README

### PR DESCRIPTION
**Pull request by jzding**
Consumer deployment should use  examples/manifests/ .
Remove the obsolete consumer manifests from examples/consumer/deployment.

### CODE REVIEW

File: examples/manifests/consumer.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ spec:
           args:
             - "--local-api-addr=127.0.0.1:9089"
             - "--api-path=/api/ocloudNotifications/v1/"
-            - "--api-addr=127.0.0.1:8089"
```
Comment (aneeshkp) [inline examples/manifests/consumer.yaml:27]: why change the port?

Comment (jzding) [reply examples/manifests/consumer.yaml:27]: to be consistent with documentation and the port used at ptp/bmer operators.

File: README.md [hunk 2]
```diff
@@ -73,12 +73,10 @@ HTTP producer example
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
-            - "--ptp-event-publisher-service-NODE_NAME.openshift-ptp.svc.cluster.local:9043"
+            - "--transport-host=http://ptp-event-publisher-service-NODE_NAME.openshift-ptp.svc.cluster.local:9043
             - "--api-port=9085"
```
Comment (aneeshkp) [inline README.md:77]: this should be 8089 too right ?

Comment (jzding) [reply README.md:77]: this is producer

**Review by nishant-parekh — APPROVED** [review]

**Pull request merged**